### ISSUE TITLE
parsing requirements

### ISSUE THREAD

**Issue by ChrisJarka** [Bible-Translation-Tools/USFM-Converter#56]
When taking the USFM file and creating the out document there should be some parsing requirements.
These parsing requirements should be the same as are used on read.bibletranslationtools.org.
Simply stated:
\f  for footnotes
\p for paragraphs
\s# for headers
\q# for quotes

Please check with Reuben S for a complete list of parsing requirements for the USFM files.

**Comment by sea-liesch** [comment]
Maybe a prompt (or an input file) to translate the word "chapter" into the target language so it can be substituted at publishing time

**Comment by sea-liesch** [comment]
Chapter and section headings have been requested

**Comment by eCyclops** [comment]
Projects which have been entered or translated in BTT-Writer should already have the word for "Chapter" in the project.

**Comment by mondele** [comment]
Indeed, the parsing engine for USFM Converter is the same as for read.bibletranslationtools.org.

I'm not aware of anywhere that a WA tool uses \s# for a section header. It may be under development, however.
